### DISCUSSION TITLE
[Suggestion] Ludum Dare game links shorten down to the game name + a little LD icon

### DISCUSSION THREAD

**Discussion by oxi-dev0** [ludumdare/ludumdare#2045] [Ideas]
It would be nice if ludum dare game links shortened down to just their name and a little Ludum dare icon, similar to how Github links get small github icons next to them, Maybe these links could also show the game icon and the ratings on hover.

E.G
`https://ldjam.com/events/ludum-dare/49/examplegame'
turns to
'(ld logo) examplegame'

**Comment by zwrawr** [comment]
Looks like there was a regression. This feature was implemented before. #1293

**Comment by oxi-dev0** [comment]
Ah i see, hopefully this gets fixed soon!
